readme.md

```diff
@@ -12,9 +12,9 @@ Most of this course will focus on the backend part. Meaning building a webserver
 
 | Week | Topic                                                     | Preparation                         | Homework                      | Lesson plan                         |
 | ---- | --------------------------------------------------------- | ----------------------------------- | ----------------------------- | ----------------------------------- |
-| 1.   | HTTP <br> Introduction to node js <br> Simple webserver   | [Preparation](week1/preparation.md) | [Homework](week1/homework.md) | [Lesson plan](week1/lesson-plan.md) |
-| 2.   | Express                                                   | [Preparation](week2/preparation.md) | [Homework](week2/homework.md) | [Lesson plan](week2/lesson-plan.md) |
-| 3.   | Database connection <br> API                              | [Preparation](week3/preparation.md) | [Homework](week3/homework.md) | [Lesson plan](week3/lesson-plan.md) |
+| 1.   | HTTP <br> Introduction to node js <br> Simple webserver   | [Preparation](week1/preparation.md) | [Homework](week1/exercise-template/readme.md) | [Lesson plan](week1/lesson-plan.md) |
+| 2.   | Express                                                   | [Preparation](week2/preparation.md) | [Homework](week2/exercise-template/readme.md) | [Lesson plan](week2/lesson-plan.md) |
+| 3.   | Database connection <br> API                              | [Preparation](week3/preparation.md) | [Homework](week3/exercise-template/readme.md) | [Lesson plan](week3/lesson-plan.md) |
```

Comment:
The updated links point to the exercises, not the homework. Is that intentional?